common/persistence/sql/sqlplugin/visibility.go

```diff
@@ -43,6 +43,7 @@ import (
 
 var (
 	ErrInvalidKeywordListDataType = errors.New("Unexpected data type in keyword list")
+	VersionColumnName = "version" 
```

Comment:
Reminder for me to add a lightweight test to verify the field name matches/exists on the row.

Reply:
I wonder if the column name `version` is too generic and cause conflicts later. Should we name something like `_version`?

Reply:
Yea, as I think about this, something like `_orderingVersion` would be more descriptive and help avoid collisions. What do you think?

Reply:
I suggested `_version` because it's the name Elasticsearch uses. I also like `_row_version`.